app/src/main/res/values-tr/strings.xml

```diff
@@ -4,26 +4,62 @@
     <!-- General/various strings -->
     <string name="menuSend">Gönder</string>
     <string name="menuSaveToSd">SD Karta kaydet</string>
-    <string name="xposed_not_active">Xposed en son sürümü şu anda aktif değildir. Şimdi Frameworku kurmak ve yeniden başlatmak istermisiniz?</string>
+    <string name="xposed_not_active">Xposed en son sürümü şu anda aktif değildir. Şimdi Frameworku kurmak ve yeniden başlatmak ister misiniz?</string>
 
     <!-- Welcome screen / main sections -->
     <string name="welcome_framework_update_available">Xposed için kullanılabilir bir güncelleştirme var.</string>
+    <string name="modules_updates_available">Modül güncellemesi mevcut</string>
 
     <!-- Navigation drawer items -->
-
+    <string name="nav_item_install">Framework</string>
+    <string name="nav_item_modules">Modüller</string>
+    <string name="nav_item_download">İndir</string>
+    <string name="nav_item_settings">Ayarlar</string>
+    <string name="nav_item_support">Destekle</string>
+    <string name="nav_item_logs">Loglar</string>
+    <string name="nav_item_about">Hakkında</string>
+    <string name="navigation_drawer_open">Navigasyon çekmecesini aç</string>
+    <string name="navigation_drawer_close">Navigasyon çekmecesini kapat</string>
+    
     <!-- Installer tab -->
     <string name="install_update">Yükle/Güncelle</string>
     <string name="uninstall">Kaldır</string>
     <string name="soft_reboot">Hızlı yeniden başlat</string>
     <string name="reboot">Yeniden Başlat</string>
+    <string name="reboot_recovery">Kurtarma moduna yeniden başlat</string>
+    <string name="not_now">Şimdi değil</string>
     <string name="reboot_failed">Y.Başlatma hatası. Lütfen telefonu normal olarak yeniden başlatın.</string>
     <string name="auto_flash_note">%s işlemi otomatik olarak yapmak istiyorsanız "TAMAM" tuşuna basın.</string>
-    <string name="reboot_confirmation">Telefonu yeniden başlatmak istiyormusunuz?</string>
+    <string name="reboot_confirmation">Telefonu yeniden başlatmak istiyor musunuz?</string>
     <string name="install_warning_title">Dikkatli olun!</string>
     <string name="install_warning">Bazı durumlarda Xposed yüklendikten sonra telefonunuz soft brick olabilir\n\nEğer \"soft brick\" veya \"bootloop\" gibi terimleri duymadıysanız\veya bunlar hakkında bilginiz yoksa Xposedi <b>YÜKLEMEYİN</b> Ya da yüklemeden önce\n\nMutlaka telefonunuzu yedekleyin.</string>
     <string name="dont_show_again">Bunu bir daha gösterme</string>
     <string name="install_known_issue">Kullandığınız rom sürümü ile (\"%s\") Bilinen bir uyumsuzluk sorunu var gibi görünüyor. Muhtemelen Xposed yüklemeniz gerekir. Daha fazla bilgi için buraya tıklayın.</string>
-
+    <string name="installer_needs_update">Xposed yükleyicisinin güncellenmesi gerekmektedir</string>
```

Comment:
This contains the app name, which shouldn't be translated. I have updated the string so that it includes a placeholder instead.

Reply:
@rovo89 Please use my translation. Please do not merge with other translation. I've completeley translated and I've tested with apktool.